pallets/space-follows/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use codec::Codec;
+use sp_std::vec::Vec;
+
+use pallet_utils::SpaceId;
+
+sp_api::decl_runtime_apis! {
+    pub trait SpaceFollowsApi<AccountId> where
+        AccountId: Codec
+    {
+        fn get_space_ids_followed_by_account(account: AccountId) -> Vec<SpaceId>;
+
+        fn filter_followed_spaces(account: AccountId, space_ids: Vec<SpaceId>) -> Vec<SpaceId>;
```

Comment:
We can implement this filtering on UI, if query all spaces the current account follows. The same applies to following accounts. So maybe no need to have `filter_followerd_spaces` and `filter_followerd_accounts` in RPC